test/integration/locate.js

```diff
@@ -0,0 +1,62 @@
+const chai = require('chai');
+const lookup = require('../../app/lib/locate');
+
+const expect = chai.expect;
+
+describe('locate', () => {
+  describe('byPostcode', () => {
+    it('Should return lat long for valid England postcode', async () => {
+      const validPostcode = 'BD24 9PT';
+      const result = await lookup.byPostcode(validPostcode);
+
+      expect(result.postcode).to.equal(validPostcode);
+      /* eslint-disable no-unused-expressions */
```

Comment:
See comment regarding the unit test specific eslint config which, if added would mean these comments could be removed.